src/io/flutter/preview/PreviewView.java

```diff
@@ -205,6 +222,15 @@ public void initToolWindow(@NotNull ToolWindow toolWindow) {
     tree.setCellRenderer(new OutlineTreeCellRenderer());
     tree.expandAll();
 
+    tree.addMouseListener(new PopupHandler() {
```

Comment:
Can we scope this to just items we know might have contributions? Nodes that represent widgets?

Reply:
Done.